tests/coverage/mcdc_if.coverage

```diff
@@ -0,0 +1,261 @@
+   LL|       |#![feature(coverage_attribute)]
+   LL|       |//@ edition: 2021
+   LL|       |//@ compile-flags: -Zcoverage-options=mcdc
+   LL|       |//@ llvm-cov-flags: --show-mcdc
+   LL|       |
+   LL|      2|fn mcdc_check_neither(a: bool, b: bool) {
+   LL|      2|    if a && b {
+                          ^0
+  ------------------
+  |---> MC/DC Decision Region (7:8) to (7:14)
+  |
+  |  Number of Conditions: 2
+  |     Condition C1 --> (7:8)
+  |     Condition C2 --> (7:13)
```

Comment:
We should replace these line numbers with `LL`, like we do for regular line numbers and branch-region line numbers.

The function that handles this is: https://github.com/rust-lang/rust/blob/4fd4797c2654977f545c9a91e2aa4e6cdbb38919/src/tools/compiletest/src/runtest.rs#L726